supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -260,12 +263,12 @@ object GenomicsFactory {
     *
     * When updating this value, also consider updating the CromwellImagesSizeRoundedUpInGB below.
     */
-  val CloudSdkImage = "gcr.io/google.com/cloudsdktool/cloud-sdk:276.0.0-slim"
+  val CloudSdkImage: String = if (config.hasPath("cloud-sdk-image-url")) { config.getString("cloud-sdk-image-url").toString } else "gcr.io/google.com/cloudsdktool/cloud-sdk:276.0.0-slim"
 
   /*
    * At the moment, the cloud-sdk:slim (727MB on 2019-09-26) and possibly stedolan/jq (182MB) decompressed ~= 1 GB
```

Comment:
I actually don't know the answer to this - do you need to do anything special for this `stedolan/jq` image?

Reply:
I believe jq only comes into play with CWL (both from memory and just now trying to trace through `WomOutputRuntimeExtractor` related code). But if one is trying to work with CWL in an environment such as this then the availability of the jq image may become an issue.